src/main/java/se/sundsvall/incidentmapper/service/IncidentService.java

```diff
@@ -194,12 +196,18 @@ private void updatePobUser(final IncidentEntity entity) {
 
 	private void updatePobComment(final IncidentEntity entity, final Issue jiraIssue) {
 		jiraIssue.getFields().getComments().stream()
-			.filter(comment -> comment.getCreated().isAfter(entity.getLastSynchronizedPob()))
+			.filter(comment -> isAfter(comment.getCreated(), entity.getLastSynchronizedPob()))
 			.filter(comment -> comment.getAuthor() != null)
 			.filter(comment -> !comment.getAuthor().getName().equals(systemUser))
 			.forEach(comment -> updatePobWithComment(entity, comment.getBody()));
 	}
 
+	private boolean isAfter(final OffsetDateTime created, final OffsetDateTime lastSynchronizedPob) {
```

Comment:
This method might be used elsewhere in this class? Rename input parameters?

Otherwise, this will also work, I suppose?

`.filter(comment -> comment.getCreated().isAfter(Optional.ofNullable(entity.getLastSynchronizedPob()).orElse(OffsetDateTime.MIN)))`